CHANGES.rst

```diff
@@ -175,6 +181,12 @@ astropy.io.votable
 astropy.modeling
 ^^^^^^^^^^^^^^^^
 
+- Major rework of modeling internals. See modeling documentation for details. 
+`<https://docs.astropy.org/en/v4.0/modeling/changes_for_4.html>`_
+
+Major change in functionality:
+Eliminates support for compound classes (but not compound instances!) [#8769]
+
```

Comment:
There are two change log entries now, one under "API changes" and one under "New Features".
Do you want to keep both? 
Also I think that blank line needs to be removed.

Reply:
It's not clear which one should be used for different comments. I'm happy to put them under only one, just point the way :-)

Reply:
You can keep the first one under "New Features" but add the PR number. And the second one can go under API changes. I hope this appeases the bot.

Reply:
Oh, and check the flake8 issue.

Reply:
already did

Reply:
Isn't the PR number already there?